core/src/main/java/com/linecorp/armeria/server/http/dynamic/Deserializers.java

```diff
@@ -0,0 +1,42 @@
+package com.linecorp.armeria.server.http.dynamic;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+
+/**
+ * Provides deserializing functionality for reflection.
+ *
+ * @see DynamicMethod
+ */
+final class Deserializers {
```

Comment:
Question: We eventually want to write something much more flexible than this, right? HTTP content type negotiation and content conversion, etc.